automation_tools/__init__.py

```diff
@@ -604,6 +604,63 @@ def setup_foreman_discovery():
     run('rm -rf {0}'.format(template_file))
 
 
+def setup_foreman_discovery_70(upstream=False):
```

Comment:
I think only package installer part is being duplicated. But it would be easier to manage in separate setups. I'll still give this a try and combine them.